gridstatus/tests/test_ercot_api.py

```diff
@@ -1,10 +1,84 @@
 import datetime
 
+import pandas as pd
 import pytest
 import pytz
 
 from gridstatus.ercot_api.api_parser import VALID_VALUE_TYPES, get_endpoints_map
-from gridstatus.ercot_api.ercot_api import hit_ercot_api
+from gridstatus.ercot_api.ercot_api import AuthenticatedErcotApi, hit_ercot_api
+
+
+class TestAuthenticatedErcotApi:
```

Comment:
All of these tests will fail without the env variables present. We going to need to find a way to get secrets into the tests. I don't think we want these variables publicly accessible in the GitHub actions config files.